place/models.py

```diff
@@ -1,59 +1,60 @@
 from django.db import models
 
 class Category(models.Model):
-    name = models.CharField(max_length=50)
+    name                        = models.CharField(max_length=50, unique=True)
 
     class Meta:
         db_table = 'categories'
 
 class Place(models.Model):
-    address                  = models.CharField(max_length=200)
-    price_per_hour           = models.IntegerField(default=10000)
-    area                     = models.FloatField(default=0)
-    floor                    = models.IntegerField(default=1)
-    maximun_parking_lot      = models.IntegerField(default=0)
-    allowed_members_count    = models.IntegerField(default=1)
-    description              = models.TextField()
-    using_rule               = models.TextField()
-    info_nearby              = models.TextField()
-    minimum_rental_hour      = models.IntegerField(default=1)
-    delegate_place_image_url = models.CharField(max_length=200)
-    surcharge_rule           = models.IntegerField(default=0)
-    user                     = models.ForeignKey('user.User', on_delete=models.CASCADE, related_name='related_place_user')
-    category                 = models.ForeignKey(Category, on_delete=models.CASCADE)
-    rating                   = models.ManyToManyField('user.User', through='Rating')
+    address                     = models.CharField(max_length=200)
+    price_per_hour              = models.IntegerField(default=10000)
```

Comment:
```suggestion
    price_per_hour              = models.DecimalField(default=10000 ,max_digits=7, decimal_places=2)
```
비용은 실수 처리해주세요. 소수점 두자리까지 주세요.

Reply:
수정하겠습니다